testsuite/tests/badly-ordered-deps/main.bytecode.reference

```diff
@@ -1,2 +1,7 @@
 File "_none_", line 1:
-Error: Wrong link order: "Lib" depends on "Main"
+Error: Wrong link order: "Lib" (lib.cmo) depends on "Main" (main.cmo)
+File "_none_", line 1:
+Error: Wrong link order: "Lib" (lib.cmo) depends on "Main" (main.cmo)
+File "_none_", line 1:
+Error: No implementations provided for the following modules:
+         "Main" referenced from "Lib" (lib.cmo)
```

Comment:
@Octachron, I've rebased the PR above your #12210. You're using `Style.as_inline_code Location.print_filename` to print filenames witch reads weird to me as filename is not code. How would you format the filename in the message above ?  Should I change it to be `from "Lib" ("lib.cmo")` ?